lib/storage/storage_test.go

```diff
@@ -713,18 +713,9 @@ func TestStorageDeleteSeries(t *testing.T) {
 	path := "TestStorageDeleteSeries"
 	s := MustOpenStorage(path, OpenOptions{})
 
-	// Verify no label names exist
-	lns, err := s.SearchLabelNames(nil, 0, 0, nil, TimeRange{}, 1e5, 1e9, noDeadline)
```

Comment:
In the cluster branch this check doesn't make any sense, since the tenants used for ingestion are non-zero. Move it to the worker check.